apps/modernization-ui/src/apps/page-builder/components/TargetQuestion/TargetQuestion.scss

```diff
@@ -1,132 +1,161 @@
 @use 'styles/borders' as borders;
 @use 'styles/colors';
 
-.target-question-tabs {
-    .tabs {
-        display: flex;
-        border-bottom: 2px solid var(--base-lighter, #dcdee0);
-        box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.09);
-        li {
-            display: inline-block;
-            cursor: pointer;
-            color: var(--base-black, #000);
-            font-family: Public Sans;
-            font-size: 1rem;
-            font-style: normal;
-            font-weight: 400;
-            line-height: normal;
-            background: colors.$base-white;
-            margin-right: 2.5rem;
-            padding-bottom: 0.62rem;
-            &.active {
-                border-bottom: 0.4375rem solid var(--primary-primary, #005ea2);
+.target-question-modal__container {
+    display: flex;
+    flex-direction: column;
+    overflow: auto;
+
+    h5 {
+        margin-top: 0;
+        margin-left: 1.5rem;
+        text-transform: none;
+    }
+
+    .target-question-tabs {
+        position: relative;
+        z-index: 1;
+        .tabs {
+            display: flex;
+            border-bottom: 2px solid var(--base-lighter, #dcdee0);
```

Comment:
Issue: Is this the same as `@extend %thin-bottom;`? If not then it should at least reference the colors:

```scss
border-bottom: 2px solid colors.$base-lighter
```